recola.spec

```diff
@@ -0,0 +1,25 @@
+### RPM external recola 1.5.0
+Source: https://recola.hepforge.org/downloads/%{n}-%{realversion}.tar.gz
+Requires: collier
+BuildRequires: cmake
+
+%define keep_archives true
+
+%prep
+%setup -q -n recola-%{realversion}
+
+%build
+rm -rf build && mkdir build
+cmake -S . -B build \
+  -DCMAKE_INSTALL_PREFIX=%{i} \
+  -Dcollier_path=${COLLIER_ROOT} \
+  -Dwith_smtests=ON
+cmake --build build %{makeprocesses}
+
+
+%install
+cmake --install build
+mkdir %{i}/cmake
+cp %{i}/lib/cmake/recolaConfig.cmake %{i}/cmake
+sed -i 's;^.*set(RECOLA_LIBRARY_DIR.*$;get_filename_component(RECOLA_LIBRARY_DIR "${CMAKE_CURRENT_LIST_DIR}/../lib" ABSOLUTE);' %{i}/cmake/recolaConfig.cmake
+sed -i 's;^.*set(RECOLA_INCLUDE_DIR.*$;get_filename_component(RECOLA_INCLUDE_DIR "${CMAKE_CURRENT_LIST_DIR}/../include" ABSOLUTE);' %{i}/cmake/recolaConfig.cmake
```

Comment:
@shimashimarin , you are copying `%{i}/lib/cmake/recolaConfig.cmake` in to `%{i}/cmake` ? is this needed? I would suggest to leave the cmake files `lib/cmake` path and run `sed` command directly on ` %{i}/lib/cmake/recolaConfig.cmake`. If you really want cmake files in top level directory then please delete the one from `%{i}/lib/cmake`

Reply:
@shimashimarin , can you please take care of it. As expected , we see build paths in `lib/cmake` files  https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-0817e1/49482/external_checks/relocate/recola.txt
```
lib/cmake/recolaConfig.cmake:set(RECOLA_LIBRARY_DIR "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/tmp/BUILDROOT/6e292a6c44125ef340634bfdd66fa7ea/opt/cmssw/el8_amd64_gcc13/external/recola/1.5.0-6e292a6c44125ef340634bfdd66fa7ea/lib")
lib/cmake/recolaConfig.cmake:set(RECOLA_INCLUDE_DIR "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/tmp/BUILDROOT/6e292a6c44125ef340634bfdd66fa7ea/opt/cmssw/el8_amd64_gcc13/external/recola/1.5.0-6e292a6c44125ef340634bfdd66fa7ea/include")

```